bindings/python/test/__init__.py

```diff
@@ -11,3 +11,40 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import pymongo
+
+
+class ClientContext:
+    def __init__(self, host='localhost', port=27017, client_kwargs=None):
```

Comment:
`client_kwargs` is never used. Can we remove it?

Reply:
Removed.